04 Angular/README.md

```diff
@@ -1,5 +1,263 @@
 # 04 Angular
 
+## We are going to create a basic Angular application, and watch what are the basic aspects that this framework use from TypeScricpt
+
+### References:
+
+* https://www.sitepoint.com/javascript-decorators-what-they-are/
+* https://www.typescriptlang.org/docs/handbook/decorators.html
+* http://www.sparkbit.pl/typescript-decorators/
+* http://blog.wolksoftware.com/decorators-reflection-javascript-typescript
+
+### 1. Create a new application using angular cli
+
+* Previous we have to install `angular-cli` [install angular-cli](https://cli.angular.io/)
+
+```bash
+$ ng new typescript-demo
+```
+
```

Comment:
Before jumping into decorators it would be a good idea to make a quick tour over the project and show that everything is typescript based